recipes/pvlib-python/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: pvlib-python
+  version: {{ version }}
+
+source:
+  fn: pvlib-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/58/3d/8431a36635ad7a06c2fa00c53769074279364bea74ab87f99389b8a90a45/pvlib-{{ version }}.tar.gz
```

Comment:
Ah, sorry, I made sure it downloaded but didn't change it for jinja. Here's the change.

```
https://pypi.io/packages/source/p/pvlib/pvlib-{{ version }}.tar.gz
```